WinUIGallery/App.xaml.cs

```diff
@@ -222,22 +179,28 @@ private async void EnsureWindow(IActivatedEventArgs args = null)
                 }
             }
 
-            NavigationRootPage rootPage = StartupWindow.Content as NavigationRootPage;
+            var rootPage = StartupWindow.Content as NavigationRootPage;
             rootPage.Navigate(targetPageType, targetPageArguments);
 
             if (targetPageType == typeof(HomePage))
             {
-                ((Microsoft.UI.Xaml.Controls.NavigationViewItem)((NavigationRootPage)App.StartupWindow.Content).NavigationView.MenuItems[0]).IsSelected = true;
+                var navItem = (NavigationViewItem) rootPage.NavigationView.MenuItems[0];
+                navItem.IsSelected = true;
             }
 
-            // Ensure the current window is active
+            // Activate the startup window.
             StartupWindow.Activate();
         }
 
+        /// <summary>
+        /// Gets the frame of the StartupWindow.
+        /// </summary>
+        /// <returns>The frame of the StartupWindow.</returns>
+        /// <exception cref="Exception">Thrown if the window doesn't have a frame with the name "rootFrame".</exception>
         public Frame GetRootFrame()
         {
             Frame rootFrame;
-            NavigationRootPage rootPage = StartupWindow.Content as NavigationRootPage;
+            var rootPage = StartupWindow.Content as NavigationRootPage;
             if (rootPage == null)
```

Comment:
Could we do 

`if (StaturWindow.Content is NavigationRootPage rootpage)` 

instead?